cmd/flux/cmd/fmt.go

```diff
@@ -0,0 +1,65 @@
+package cmd
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/ast/astutil"
+	"github.com/influxdata/flux/parser"
+	"github.com/spf13/cobra"
+)
+
+// fmtCmd represents the fmt command
+var fmtCmd = &cobra.Command{
+	Use:   "fmt",
+	Short: "Format a Flux script-file",
+	Long:  "Format a Flux script-file (flux fmt [-w] <script-file>)",
+	Args:  cobra.MinimumNArgs(1),
+	RunE:  format,
+}
+
+var writeResultToSource bool
+
+func init() {
+	rootCmd.AddCommand(fmtCmd)
+	fmtCmd.Flags().BoolVarP(&writeResultToSource, "write-result-to-source", "w", false, "write result to (source) file instead of stdout")
+}
+
+func format(cmd *cobra.Command, args []string) error {
+	script := args[0]
+
+	fromFile, err := ioutil.ReadFile(script)
+	if err != nil {
+		return err
+	}
+
+	pkg := parser.ParseSource(string(fromFile))
+	if ast.Check(pkg) > 0 {
+		return fmt.Errorf("unexpected error: %s , filename: %s", ast.GetError(pkg), script)
```

Comment:
If I could offer some editorial guidance, it would be good to format errors here like this:
```
parse error: foo.flux|10:20-10:21: expected RPAREN
```
Or something like that. Note that errors returned from parsing (I believe) may include the file, line  number and column when they get formatted already. However errors format their source location info is fine, but the name of the file should appear first.

Reply:
Error from the parser doesn't contain the filename, that is why I have added that. 
Sample error (replaced unexpected with parse)- 

parse error: loc 16:47-16:48: invalid statement @16:47-16:48: : , filename: stdlib/array/array.flux